db/migrate/20160719193732_add_hidden_method_to_design_methods.rb

```diff
@@ -0,0 +1,5 @@
+class AddHiddenMethodToDesignMethods < ActiveRecord::Migration
+  def change
+    add_column :design_methods, :hidden_method, :boolean, :default => false
```

Comment:
Why did you settle on `hidden_method` rather than just `hidden`? Not sure I understand the sequence of migrations here.